src/ds/json/modify.md

```diff
@@ -0,0 +1,109 @@
+Modifying JSON documents is straightforward using a combination of JSONPath expressions and Redis's JSON command set.
+
+Here are some examples.
+
+## Extend documents
+
+```redis Add a new key:value pair to an existing document
```

Comment:
Add a new member (name-value pair) to an existing object (in this case the root of the document is an object)
Rename `newkey` to `newmember`.